cypress/e2e/landing.cy.js

```diff
@@ -0,0 +1,32 @@
+describe('landing page loads', () => {
+    beforeEach(() => {
+      cy.visit('/landing')
+      cy.injectAxe();
+    })
+  
+    it('displays the landing page', () => {
+      cy.url().should("contains", "/landing");
+    })
+
+    it('should display the button for no ESRF',()=>{
+        cy.contains(`I don't have my file number (ESRF)`).should('be.visible')
+    })
```

Comment:
ok makes sense. I was just hesitant because there was no id on those elements and they were different elements (one was a button and the other was an a tag) I will make the appropriate changes.

Reply:
there were a few spots I saw them; even for button clicks.

Reply:
Added id instead of text to reference